pkg/sql/sqlbase/column_type_properties.go

```diff
@@ -51,11 +52,23 @@ func AdjustValueToColumnType(
 			sv = v.Contents
 		}
 
+		if typ.Oid() == oid.T_bpchar {
```

Comment:
This is everywhere -- is there not a function we can pull this and other common string logic out into?

Reply:
everywhere is slightly different, unfortunately :(